.github/workflows/website.yml

```
@@ -48,30 +68,77 @@ jobs:
       - uses: bahmutov/npm-install@v1
         env:
           PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1
-      - run: npm run build -w packages/website
+      - run: touch .env
+      - run: |
+          (
+            npm -w '@web3-storage/website' run ci:envfile --silent \
+            | sed "s/^NEXT_PUBLIC_MAGIC=$/NEXT_PUBLIC_MAGIC=$NEXT_PUBLIC_MAGIC/" \
+            | sed "s/^MAGIC_SECRET_KEY=$/MAGIC_SECRET_KEY=$MAGIC_SECRET_KEY/" \
+            ;
+            echo 'NEXT_PUBLIC_MAGIC_TESTMODE_ENABLED="1"';
+          ) > .env
         env:
-          NEXT_PUBLIC_API: http://localhost:8787
-          NEXT_PUBLIC_ENV: test-e2e
           NEXT_PUBLIC_MAGIC: ${{ secrets.STAGING_MAGIC_PUBLIC_KEY }}
+          MAGIC_SECRET_KEY: ${{ secrets.STAGING_MAGIC_SECRET_KEY }}
+      - name: load db schema
+        run: |
+          npm -w '@web3-storage/db' run start
+          npm -w '@web3-storage/db' run load-schema
+          npm -w '@web3-storage/db' run stop
```

nit: elsewhere in our workflows we're using the workspace path like:
```suggestion
          npm run start -w packages/db
          npm run load-schema -w packages/db
          npm run stop -w packages/db
```

---

ok I find it less readable because 1) it helps me to first know what package the thing will run in and 2) the '-w' line up better in the text editor if they go first. however, I will use the suggested format for the sake of consistency